index.md

```diff
@@ -7,8 +7,11 @@ redirect_from: /lang/en/
 
 ## Summary
 
-As a software developer, I want to squash feature branches onto `master` and write
-a standardized commit message while doing so.
+As a software developer, I want to create a readable commit history writing standardized commit messages to make easier and explicit what changes have been applied to a project.
```

Comment:
> As a software developer

The first part of the first sentence still feels exclusionary to non Software Developers.

I'm not sure calling out a specific profession adds to the rest of the sentence. I feel the rest of the sentence can stand on its own.

Reply:
I think the summary should be more to the point about the actual work that someone is undertaking when writing conventional commit messages.

The point of the summary is to tell people what they're actually doing, not to sell them on the value of conventional commit messages -- here's a suggestion:

```When writing software use a standardized commit message, such that the
commit history conveys meaning to developers and can be more easily
consumed by tools.```